File: mc_server/src/main/java/com/example/mc_server/entity/Companion.java
package com.example.mc_server.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class Companion {
    private Long id;
    private String name;
    private String avatar;
    private String sex;
    private Integer age;
    private String mobile;
    private Integer active;
    private Integer createUserId;
}

File: mc_h5/src/types/companion.ts

//陪护员请求参数类型
export type PhotoList = { 
  name: string;
  url: string;
};

//陪护员列表结构
export type CompanionList = {
  createTime?: string;  
  id: number;
  name: string;
  mobile: string;
  avatar?: string;
  sex: string;
  age: number;
  active?: number;
};

